pkg/webhooks/validation/server/server.go

```diff
@@ -273,14 +263,10 @@ func (wh *Webhook) Run(stopCh <-chan struct{}) {
 		case <-keyCertTimerC:
 			keyCertTimerC = nil
 			wh.reloadKeyCert()
-		case event, more := <-wh.keyCertWatcher.Events(wh.keyFile):
-			if more && isModify(event) && keyCertTimerC == nil {
-				keyCertTimerC = time.After(watchDebounceDelay)
-			}
-		case event, more := <-wh.keyCertWatcher.Events(wh.certFile):
-			if more && isModify(event) && keyCertTimerC == nil {
-				keyCertTimerC = time.After(watchDebounceDelay)
-			}
+		case <-wh.keyCertWatcher.Events(wh.keyFile):
```

Comment:
The `keyCertTimerC == nil` check needs to be retained to preserve the debounce behavior.

Reply:
Thanks, fixed